src/write_outputs/write_outputs.jl

```diff
@@ -108,7 +108,7 @@ function write_outputs(EP::Model, path::AbstractString, setup::Dict, inputs::Dic
 
 
 	# Output additional variables related inter-period energy transfer via storage
-	if setup["OperationWrapping"] == 1 && !isempty(inputs["STOR_LONG_DURATION"])
+	if !isempty(inputs["STOR_LONG_DURATION"])
```

Comment:
If there's only one representative period, STOR_LONG_DURATION will be empty.

Reply:
Make sense.